services/QuillLMS/client/app/bundles/Shared/components/shared/dataTable.tsx

```diff
@@ -264,7 +264,11 @@ export class DataTable extends React.Component<DataTableProps, DataTableState> {
   }
 
   renderActions(row) {
-    if (!row.actions) { return }
+    if (!row.actions) {
+      return(
+        <td className="data-table-row-section actions-section" key={row.id}><p className="no-action">--</p></td>
+      )
+    }
```

Comment:
confirmed with Jack that we want this universal change to display dashes when no action is present